### PR TITLE
CFY-4625 add composer installation docs

### PR DESCRIPTION
**Pull request by nir0s**

### CODE REVIEW

File: content/composer/installation.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+layout: bt_wiki
+title: Installation
+category: Docs
+draft: false
+weight: 10
+
+---
+
+Installing Cloudify's Blueprint Composer is currently done via a script.
```
Comment (idanmo) [inline content/composer/installation.md]: dot at the end of the sentence is necessary?

File: content/composer/installation.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+```
+
+And follow the instructions to run it.
+
+Note that if the `python` executable in your path is not python2.7 by default, you can run the script using `python2` instead:
+
+``shell
+sudo python2.7 get-cloudify-composer.py
+```
+
```
Comment (idanmo) [inline content/composer/installation.md]: Perhaps also add the instructions for starting it?

File: content/composer/installation.md (hunk 1 above)
Comment (nir0s) [inline content/composer/installation.md]: um? it's a sentence :0

File: content/composer/installation.md (hunk 2 above)
Comment (nir0s) [inline content/composer/installation.md]: sure.
also.. for uninstallation

File: content/composer/installation.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+layout: bt_wiki
+title: Installation
+category: Docs
+draft: false
+weight: 10
+
+---
+
+Installing Cloudify's Blueprint Composer is currently done via a script.
+which can be downloaded from [here](http://gigaspaces-repository-eu.s3.amazonaws.com/org/cloudify3/get-cloudify-composer.py).
```
Comment (uric) [inline content/composer/installation.md]: Needs to point to our download page.

File: content/composer/installation.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+
+# Prerequisites
+
+The script requires that you have:
+
+* Python 2.7.x
+* pip 1.5+
+* Virtualenv 12+
+* Sudo privileges
+* An internet connection.
```
Comment (uric) [inline content/composer/installation.md]: Which OS?

**Pull request merged**